src/main/docs/guide/cloud/cloudConfiguration/distributedConfigurationAwsParameterStore.adoc

```diff
@@ -12,7 +12,7 @@ You can configure shared properties by going into the AWS Console -> System Mana
 
 Micronaut will use a hierarchy to read the configuration values, and supports `String`, `StringList`, and `SecureString` types.
 
-You can make environment specific configurations as well by including the environment name after an underscore `_`. For example if your `micronaut.application.name` setting is set to `helloworld` then providing configuration values under `helloworld_test` will be applied only to the `test` environment.
+You can make environment specific configurations as well by including the environment name after an underscore `_`.  Providing configuration values under `application_test` will be applied only to the `test` environment. Please note currently environment specific values of application name are not supported (i.e. `helloworld_test`).
```

Comment:
Should `helloworld_test` be supported? Is there some technical limitation?

Reply:
#https://github.com/micronaut-projects/micronaut-aws/issues/3

I am able to access the configuration from AWS parameter store now,

I removed the explicit root hierarchy path, and named a parameter in AWS as:
config/${applicationName}/db/dbname
and accessed it using @Property(name = "db.dbname")

Reply:
I think it should. I did testing on what was currently released and working in 1.1.0. I think maybe a previous PR might have broken that but I'm not 100% but I couldn't get it to work.